api/config/config_stable.json

```diff
@@ -12,7 +12,7 @@
     "xAppIdValue": "stable-AoU-RW",
     "timeoutInSeconds": 40,
     "lenientTimeoutInSeconds": 180,
-    "jupyterDockerImage": "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:2.2.7",
+    "jupyterDockerImage": "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:2.2.12",
```

Comment:
stable and staging also use Terra Prod, so they will also need to wait for the Terra release

Reply:
Leo PR is merged, most likely the change would be released to prod tomorrow or some time this week. And we already cut release for this week. Most likely it should be fine and staging & stable is actively used. I might will just have staging /stable in this pr and do preprod and prod in seperate PR as long as we verify it works in other envs.